Description: correct some typos in man page
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Forwarded: not-needed
Last-Update: 2014-10-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/doc/xsane.man
===================================================================
--- trunk.orig/doc/xsane.man
+++ trunk/doc/xsane.man
@@ -57,38 +57,38 @@ To run
 under the
 .BR gimp (1),
 you should at first make sure that xsane is
-compiled with gimp support by entering "xsane -v" on a shell.
+compiled with gimp support by entering "xsane \-v" on a shell.
 If xsane is compiled with gimp support then simply set a
 symbolic link from the xsane-binary to one of the
 .BR gimp (1)
-plug-ins directories.  For example, for gimp-1.0.x the command
+plug-ins directories.  For example, for gimp\-1.0.x the command
 .PP
 .RS
-ln -s @BINDIR@/xsane ~/.gimp/plug-ins/
+ln \-s @BINDIR@/xsane ~/.gimp/plug\-ins/
 .RE
 .PP
 for gimp 1.2.x the command:
 .PP
 .RS
-ln -s @BINDIR@/xsane ~/.gimp-1.2/plug-ins/
+ln \-s @BINDIR@/xsane ~/.gimp\-1.2/plug\-ins/
 .RE
 .PP
 and for gimp 2.0.x the command:
 .PP
 .RS
-ln -s @BINDIR@/xsane ~/.gimp-2.0/plug-ins/
+ln \-s @BINDIR@/xsane ~/.gimp\-2.0/plug\-ins/
 .RE
 .PP
 adds a symlink for the
 .B xsane
-binary to the user's plug-ins directory.  After creating this symlink,
+binary to the user's plug\-ins directory.  After creating this symlink,
 .B xsane
 will be queried by
 .BR gimp (1)
 the next time it's invoked.  From then on,
 .B xsane
-can be invoked through "Xtns->XSane->Device dialog..." (gimp-1.0.x) or through
-"File->Acquire->XSane->Device dialog..." (gimp-1.2.x and 2.0.x) menu entry.
+can be invoked through "Xtns->XSane->Device dialog..." (gimp\-1.0.x) or through
+"File->Acquire->XSane->Device dialog..." (gimp\-1.2.x and 2.0.x) menu entry.
 
 You'll also find that the "Xtns->XSane" or "File->Acquire->XSane" menu contains
 short-cuts to the SANE devices that were available at the time the
